lib/src/Uncertainty/Algorithm/MetaModel/Kriging/openturns/KrigingAlgorithm.hxx

```diff
@@ -53,6 +53,11 @@ public:
                     const CovarianceModel & covarianceModel,
                     const Basis & basis);
 
+  /** Constructor */
+  KrigingAlgorithm(const Sample &inputSample,
+                   const Sample &outputSample,
+                   const CovarianceModel &covarianceModel);
```

Comment:
I think its enough to make basis optional in the previous ctor:
```const Basis & basis = Basis()```
then it looks like GLM will handle this

Reply:
ok